tests/test_message/test_message_create.py

```diff
@@ -61,21 +69,59 @@ def _add_and_return(message, exchange=None):
         yield sent
 
 
+@pytest.mark.parametrize(
+    "kind, schema, username, request_data, db_service, request_headers",
+    [
+        pytest.param(
+            "github",
+            GitHubMessageV1,
+            "dummy-fas-username",
+            "github",
+            "github",
+            "github",
+            id="GitHub - Sending data and successfully creating message",
```

Comment:
You only need "GitHub" here now, the rest is in the docstring. This also applies to other places in this file.